mathics/builtin/strings.py

```diff
@@ -42,6 +43,36 @@
 }
 
 
+alphabet_descriptions = {
+    "English": {
+        "Lowercase": "abcdefghijklmnopqrstuvwxyz",
+        "Uppercase": "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    "Spanish": {
+        "Lowercase": "abcdefghijklmnñopqrstuvwxyz",
```

Comment:
WL Aphabets omit any accented characters such as those in Spanish above an "e", but leave those tilde for "n"?  Similarly for an umlaut for Germain?

Reply:
No! This is the "Eñe" and is a letter very important for Spanish speakers! https://en.m.wikipedia.org/wiki/%C3%91
:)